src/js/pages/services/ServicesTab.js

```diff
@@ -256,6 +256,8 @@ var ServicesTab = React.createClass({
     // Find item in root tree and default to root tree if there is no match
     let item = DCOSStore.serviceTree.findItemById(id) || DCOSStore.serviceTree;
 
+    let service = new Service({id: item.getId().replace(/^(\/.+)$/, '$1/')});
```

Comment:
I don't feel comfortable with instantiating a new service on every render call, just to pass the group/context. Let's introduce a prop to pass the context into the modal.